source/stylesheets/_variables.sass

```diff
@@ -23,6 +23,8 @@ $inventory-bar-color: #{"{{ theme.inventory_bar_color }}"}
 
 $primary-font: #{"{{ theme.primary_font | font_family }}"}
 $secondary-font: #{"{{ theme.secondary_font | font_family }}"}
+$input-border: #666666
+$input-background: #F7F7F7
```

Comment:
Curious where these hard-coded values are coming from, should they reference theme color settings instead?

Reply:
@nickendle Julia specified them in the story. Is there another place I can get or store these colors? I am not sure I understand what you mean by theme color settings-- is that custo?

For reference, they style this part of the cart:

This input:
<img width="85" alt="Screen Shot 2019-11-22 at 11 21 55 AM" src="https://user-images.githubusercontent.com/13512356/69442295-51265980-0d1a-11ea-8569-79ca7417f5be.png">


In context:
<img width="737" alt="Screen Shot 2019-11-22 at 11 21 58 AM" src="https://user-images.githubusercontent.com/13512356/69442294-51265980-0d1a-11ea-94cb-c838d3886821.png">

Reply:
@hwinn4 we typically want to avoid any hard-coded color values in themes and instead use values defined in the custo area. there's already styles defined for input+textarea in layout.css, can it inherit those styles?

Reply:
We resolved this in basecamp-- these color changes will be removed and we will continue using the default theme colors.
https://3.basecamp.com/3864451/buckets/14176167/chats/2148055898@2223969335